plugins/web/ui/src/routes/status/+page.svelte

```diff
@@ -52,12 +52,12 @@
     { header: 'Percentage', accessor: 'percentage' }
   ];
 
-  const getEventsRows = (events: Events) => {
+  const getEventsRows = (events: Events, total_events: number) => {
```

Comment:
```suggestion
  const getEventsRows = (events: Events, totalEvents: number) => {
```

JS/TS typically uses camelCase.